src/index.html

```diff
@@ -8,9 +8,46 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form form--big"
+      action="/"
+      method="post"
+      data-qa="big"
+      >
```

Comment:
Still not fixed from the previous review
```suggestion
    <form
      class="form form--big"
      action="/"
      method="post"
      data-qa="big"
    >
```